v3/src/components/graph/components/graph-component.tsx

```diff
@@ -65,23 +65,24 @@ export const GraphComponent = observer(function GraphComponent({ tile }: ITileBa
   if (!graphModel) return null
 
   return (
-    <DataSetContext.Provider value={data}>
-      <InstanceIdContext.Provider value={instanceId}>
-        <GraphLayoutContext.Provider value={layout}>
-          <AxisLayoutContext.Provider value={layout}>
-            <GraphContentModelContext.Provider value={graphModel}>
-              <AxisProviderContext.Provider value={graphModel}>
-                <Graph
-                  graphController={graphController}
-                  graphRef={graphRef}
-                  pixiPointsRef={pixiPointsRef}
-                />
-              </AxisProviderContext.Provider>
-              <AttributeDragOverlay activeDragId={overlayDragId} />
-            </GraphContentModelContext.Provider>
-          </AxisLayoutContext.Provider>
-        </GraphLayoutContext.Provider>
-      </InstanceIdContext.Provider>
-    </DataSetContext.Provider>
+    pixiPointsRef.current &&
```

Comment:
Should be able to remove this now, right?

Reply:
Weird. I swear I removed that previously, but I must not have committed the change somehow. 

I retesting things after removing it again and the bug is still fixed.